content/python/concepts/built-in-functions/terms/exec/exec.md

```diff
@@ -0,0 +1,98 @@
+--
+Title: 'exec()'
+Description: 'Executes code object or string containing python code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+Returns ``None``.
+
+## Syntax
+
+```py
+exec(object, globals=None, locals=None)
+```
+
+* ```object```: String or code object
+* ```globals```: (optional) A dictionary containing global variables. If not specified, defaults to ```None`. 
+* ```locals```: (optional) A dictionary containing global variables. If not specified, defaults to ```None```.
+
+## Example 1
+
+Use `exec()` to parse and execute Python code contained in the string `code`:
+
+```codebyte/python
```

Comment:
````suggestion
```py
````
We generally only have one codebyte at the end of the entry.

Instead, add a ` ```shell ` block with the output after the example along with an introductory sentence. Like this:

````
This example results in the following output:

```shell
Hello, Codecademy!
```
````
```